src/WormholeJoin.sol

```diff
@@ -75,6 +75,16 @@ contract WormholeJoin {
         uint248 pending;
     }
 
+    // --- Errors ---
+    error NotAuthorized(address sender, uint256 wards);
+    error FileUnrecognizedParam(bytes32 what);
+    error NotAllowedBiggerInt256(uint256 data);
+    error IncorrectDomain(bytes32 targetDomain, bytes32 domain);
+    error MaxFeeExceeded(uint256 fee, uint256 maxFee);
+    error AlreadyBlessed(bytes32 hashGUID, bool blessed);
```

Comment:
This can be reduced to just the hash I think: `AlreadyBlessed(bytes32 hashGUID);`

Reply:
done in 0172295